.github/workflows/windows-meson-msys2.yml

```diff
@@ -0,0 +1,136 @@
+name: Windows Meson MSYS2 builds
```

Comment:
Suggest calling this 'MSYS2 builds', to be a bit easy for casual scanning over the actions page.

![2021-11-19_18-18](https://user-images.githubusercontent.com/1557255/142711076-b7abf4c0-ad24-421d-85e0-730afd6865e1.png)

Reply:
Ah, good idea.